stix_shifter_modules/error_test/stix_translation/query_translator.py

```diff
@@ -0,0 +1,30 @@
+from stix_shifter_utils.modules.base.stix_translation.empty_query_translator import EmptyQueryTranslator
+import re
+
+START_STOP_PATTERN = "\s?START\s?t'\d{4}(-\d{2}){2}T\d{2}(:\d{2}){2}(\.\d+)?Z'\sSTOP\s?t'\d{4}(-\d{2}){2}T(\d{2}:){2}\d{2}.\d{1,3}Z'\s?"
+
+ERROR_TYPE_PARSE_EXCEPTION = 'parse_exception'
+ERROR_TYPE_TRANSFORM_EXCEPTION = 'transform_exception'
+
+class QueryTranslator(EmptyQueryTranslator):
+
+    def parse_query(self, data):
+        print('**IN PARSE QUERY')
+        print(self.options.get('error_type'))
```

Comment:
you can replace print statements with our logger utility `from stix_shifter_utils.utils import logger` unless you have specific case for print statements.

Reply:
I missed this one. Will remove